styles/styles.css

```diff
@@ -0,0 +1,214 @@
+body {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  font-family: Helvetica;
+}
+
+header {
+  background-image: url(https://creator-assets.wework.com/wp-content/uploads/2017/05/Helpful-Tips-for-Your-StartUp-Office-Design.jpg);
+  background-size: cover;
+  background-position: centertop;
+  background-repeat: no-repeat;
+  text-shadow: 1px 1px 4px rgba(0,0,0,0.8);
+}
+
+
+ul {
+  padding: 0;
+  list-style: none;
+  margin: 0;
+}
+
+nav ul {
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr 1fr;
+  list-style: none;
+  color: white;
+  padding: 2rem 0 0 0;
+}
+
+nav ul li {
+  display: flex;
+  justify-content: center;
+  font-size: 0.8em;
+  font-weight: bold;
+}
+
+nav ul li:first-of-type {
+  grid-column: 1 / 2;
+}
+
+nav ul li:nth-of-type(2) {
+  grid-column: 3 / 4;
+}
+
+nav ul li:last-of-type {
+  grid-column: 4 / 5;
+}
+
+header section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3.5em;
+  height: 20rem;
+  justify-content: center;
+  color: white;
+}
+
+header section a {
+  border-radius: 10%;
```

Comment:
I think using px instead of % for the rounded corners would fix the button border image.